docs/DeveloperGuide.md

```diff
@@ -478,317 +478,330 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
+* 2a. Invalid Consultation Index
+  * 2a1. TAHub shows an error message stating that the Consultation Index is invalid.
+    <br>
+    Use case ends.
+* 2b. Invalid Student Index
+  * 2b1. TAHub shows an error message stating that the Student Index is invalid.
+    <br>
+    Use case ends.
+* 2c. Student Name not Found in Student List.
+  * 2c1. TAHub shows an error message stating that the Student does not exist.
+    <br>
+    Use case ends.
+* 2d. Student is not in consultation
+  * 2d1. TAHub shows an error message stating that the Student is not in the consultation.
+    <br>
+    Use case ends.
+
+
+**<u>Use case: UC12 - Delete Consultation</u>**
+
+**Precondition:** The Consultation exists in TAHub.
 
-* 2a. Invalid consultation index
-    * 2a1. System shows error message
-    * 2a2. Use case resumes from step 2
-
-* 3a. Student not found in system
-    * 3a1. System shows error message identifying missing student
-    * 3a2. Use case resumes from step 3
-
-* 4a. Student not in consultation
-    * 4a1. System shows error message that student is not in consultation
-    * 4a2. Use case resumes from step 3
-
-**<u>Use case: UC11 - Export Student Data</u>**
-
-**MSS**
-
-1. TA enters export command with desired filename
-2. System validates filename
-3. System creates CSV file in data directory
-4. System copies file to home directory
-5. Success message shows number of students exported
+**MSS:**
+1. User requests to delete a specific consultation by providing the necessary details (Consultation Index).
+2. TAHub verifies the Consultation Index.
+3. TAHub deletes the consultation at the index in the Consultation List.
+<br>
+Use case ends.
 
 **Extensions**
+* 2a. Invalid Consultation Index
+  * 2a1. TAHub shows an error message stating that the Consultation Index is invalid.
+  <br>
+  Use case ends.
 
-* 2a. Invalid filename
-    * 2a1. System shows error message about invalid characters
-    * 2a2. Use case resumes from step 1
-
-* 3a. File already exists
-    * 3a1. System shows error message suggesting force flag
-    * 3a2. Use case resumes from step 1
-
-* 3b. Directory creation fails
-    * 3b1. System shows error message about directory creation
-    * 3b2. Use case ends
 
 **<u>Use case: UC12 - Export Consultation Data</u>**
 
-**MSS**
-
-1. TA enters exportconsult command with desired filename
-2. System validates filename
-3. System creates CSV file with consultation data
-4. System copies file to home directory
-5. Success message shows number of consultations exported
-
-**Extensions**
-
-* [Same extensions as UC11]
-
-**<u>Use case: UC13 - Import Student Data</u>**
+**MSS:**
+1. User requests to export consultation data with desired filename.
+2. TAHub validates filename.
+3. TAHub creates CSV file with consultation data.
+4. TAHub copies file to home directory.
+5. Success message shows number of consultations exported.
+<br>
+Use case ends.
 
-**MSS**
+**Extensions:**
+* [Same extensions as UC6]
```

Comment:
No idea if this is allowed, but well done if it is.

Reply:
This was done by @yhanyi. I've no idea either, but I left it as it is.